src/rpc/rawtransaction.cpp

```diff
@@ -672,88 +672,13 @@ UniValue combinerawtransaction(const JSONRPCRequest& request)
     return EncodeHexTx(mergedTx);
 }
 
-UniValue signrawtransaction(const JSONRPCRequest& request)
+UniValue sign_transaction(CMutableTransaction& mtx, const UniValue& prevTxsUnival, CBasicKeyStore *keystore, bool tempKeystore, const UniValue& hashType)
```

Comment:
non blocking nit: if this stuff is being renamed we should follow snake case

Reply:
I believe you meant CamelCase, not snake_case. I changed it to CanelCase.

Reply:
referring to "variable and namespace names are all lowercase, and may use _ to separate words (snake_case)." in https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md

Reply:
Ah. Most of these aren't renamed (I think).

Reply:
Please no more renames! :)